health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/BoundaryRelationCreate.js

```
@@ -0,0 +1,682 @@
+                <React.Fragment>
+                    {firstPage && (
+                        <div>
+                            <Card type={"primary"} variant={"viewcard"} className={"example-view-card"}>
+                                <div style={{ display: "flex", justifyContent: "space-between" }}>
+                                    <div className="hierarchy-boundary-heading">{t("CREATE_BOUNDARY_HIERARCHY")}</div>
+                                </div>
+                                <div style={{ height: "1.5rem" }}></div>
+                                <div>
+                                    {t("MSG")}
+                                </div>
+                                <div style={{ height: "2rem" }}></div>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)